docs/source/extend_kedro/plugins.md

```diff
@@ -78,7 +77,54 @@ Plugins may also add commands to the Kedro CLI, which supports two types of comm
 
 ## Suggested command convention
 
-We use the following command convention: `kedro <plugin-name> <command>`, with `kedro <plugin-name>` acting as a top-level command group. This is our suggested way of structuring your plugin bit it is not necessary for your plugin to work.
+We use the following command convention: `kedro <plugin-name> <command>`, with `kedro <plugin-name>` acting as a top-level command group. This is our suggested way of structuring your plugin but it is not necessary for your plugin to work.
+
+## Advanced: Lazy loading of plugin commands
+
+If you are developing a plugin with a large set of CLI commands or with certain large libraries that are slow to import for only a small subset of commands, you can consider using lazy loading of these commands to improve the performance of the plugin as well as the overall performance of Kedro CLI. To do this, you can follow [the instructions on the
```

Comment:
Maybe break up the first sentence to make it slightly easier to read. For example:

"If you are developing a plugin with a large set of CLI commands or with certain large libraries that are slow to import for only a small subset of commands, consider lazy loading those commands. This can significantly improve the performance of both your plugin and the Kedro CLI overall."